hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueContainer.java

```diff
@@ -60,8 +64,9 @@
  * </ul>
  */
 @SuppressWarnings("checkstyle:methodcount")
-public class QueueContainer implements IdentifiedDataSerializable {
+public class QueueContainer implements IdentifiedDataSerializable, SplitBrainAwareDataContainer<Long, Data, QueueItem> {
 
+    private static final int ID_PROMOTION_OFFSET = 100000;
```

Comment:
This can now be removed and `CollectionContainer#ID_PROMOTION_OFFSET` used instead